application/liveness/liveness_test.go

```diff
@@ -0,0 +1,27 @@
+package liveness
+
+import (
+	"fmt"
+	"testing"
+	"time"
+	"os"
+)
+
+func TestBasic(t *testing.T) {
+	os.Setenv("PHANTOM_SUBNET_LOCATION", "../lib/test/phantom_subnets.toml")

```

Comment:
Because these tests kick off a full zmap scan we should make sure they aren't running every time we run package tests. We can do this by adding a check for an environment variable:

```
func TestBasic(t *testing.T) {
	if os.Getenv("MEASUREMENTS") != "1" {
		t.Skip("skiping long running measurement based tests")
	}
	os.Setenv("PHANTOM_SUBNET_LOCATION", "../lib/test/phantom_subnets.toml")
      ....
```

it might also be nice to have a comment that says how to run this:
```
// To run the measuremest commands set the environment variable when running go test
//     $ MEASUREMENTS=1 go test -v
```